webpack.config.js

```diff
@@ -143,4 +135,20 @@ if (!IS_DEV) {
   );
 }
 
+const files = glob.sync(process.cwd() + '/src/*.html');
+
+files.forEach(file => {
+  config.plugins.push(
+    new HtmlWebPackPlugin({
+      template: file,
+      favicon: path.resolve(__dirname, './src/public/icon.ico'),
```

Comment:
May need the option [`filename`](https://github.com/jantimon/html-webpack-plugin#options) here.